src/Analyzers/CSharp/CodeFixes/UseDeconstruction/CSharpUseDeconstructionCodeFixProvider.cs

```diff
@@ -120,10 +120,7 @@ private ForEachVariableStatementSyntax CreateForEachVariableStatement(INamedType
             // However, convert the existing declaration over to a "var (x, y)" declaration or (int x, int y)
             // tuple expression.
             return SyntaxFactory.ForEachVariableStatement(
-#if !CODE_STYLE // ForEachStatementSyntax.AttributeLists is not available in the version of Microsoft.CodeAnalysis used by CodeStyle layer
-                // https://github.com/dotnet/roslyn/issues/41462#issuecomment-595893953 tracks removing these conditional directives.
                 forEachStatement.AttributeLists,
-#endif
```

Comment:
nice.  how are you finding this.  are you just going to all CODE_STYLE locations and trying to remove them?

Reply:
Yep, exactly.